include/sentry.h

```diff
@@ -1362,6 +1362,12 @@ SENTRY_API sentry_user_consent_t sentry_user_consent_get(void);
  */
 SENTRY_API sentry_uuid_t sentry_capture_event(sentry_value_t event);
 
+/**
+ *  Allows capturing independently created minidumps.
```

Comment:
Given that this interface has no peers in the standard SDK interfaces, a quick spec of what happens is in order here, especially that:

* it generates a dummy event with log-level `FATAL` and
* this generated event is processed like any other event, including attachments, scope application, and before-send hook.

Reply:
OK, updated the doc